api/envoy/api/v2/rds.proto

```diff
@@ -44,7 +45,25 @@ service RouteDiscoveryService {
   }
 }
 
-// [#comment:next free field: 9]
+// * HTTP :ref:`router filter <config_http_filters_router>`is
+// Virtual Host Discovery Service (VHDS) is used to dynamically update the list of virtual hosts for
+// a given RouteConfiguration. If VHDS is configured a virtual host list update will be triggerred
+// during an http request processing if a route for the request cannot be resolved. The
+// *resource_names_subscribe* field in :ref:`DeltaDiscoveryRequest
+// <envoy_api_msg_DeltaDiscoveryRequest>` contains a list of virtual host names or aliases to track.
+// The contents of an alias would be the contents of a *host* or *authority* header used to make an
+// http request. An xDS server will match an alias to a virtual host based on the content of
+// :ref:`VirtualHosts' <envoy_api_msg_route.VirtualHost>` *domains* field. The
```

Comment:
Nit: you can actually link directly to fields, grep for `envoy_api_field_` to see how.

Reply:
done